### DISCUSSION TITLE
mlutils para repriducir el NOTES 04.02 - DATA CLEANING.ipynb

### DISCUSSION THREAD

**Discussion by malalalalala** [rramosp/ai4eng.v1#291] [04 - GESTIÓN DE DATOS]
Buenas noches, quiero incluir en el proyecton finla el análisis que se hace en ese google colab pero al parecer la librería mlutils es una librería personalizada que hizo el profe o que está local solamente para él. Hay alguna manera de poder replicar esto? Muchas gracias 
![image](https://github.com/rramosp/ai4eng.v1/assets/59240499/081fb688-ce6e-4dce-8795-2de7549f72a2)

**Comment by Felipe-RA** [comment]
Saludos @malalalalala 

Sobre  la libreria `mlutils.py`, puedes encontrar el código fuente  en este mismo repositorio bajo [esta dirección](https://github.com/rramosp/ai4eng.v1/blob/da4d9f557c0374ba17b9d75e50668bdb0a1e4a62/content/local/lib/mlutils.py)

Probablemente olvidaste correr la celda que importa esta libreria (`from local.lib import mlutils`) y por eso no te resuelve el import

Hasta pronto!
